server/src/main/resources/application.yml

```diff
@@ -70,4 +70,4 @@ flyway:
   enabled: enabled
   datasources:
     default:
-      locations: classpath:db
+      locations: classpath:db/common
```

Comment:
why did we change the class path ?

Reply:
This isn't a change to the classpath; it's a change to where on the classpath we look for scripts. This is done so we can split scripts into different directories under db to control which get run under what circumstance.

Reply:
...But should not be part of this story. A different branch leaked onto this one.

Reply:
I guess that was more of my concern that why it was changed as part of this story :)